src/Storage/Migration/v0.00/03-setup-functions.sql

```diff
@@ -284,18 +284,6 @@ RETURN QUERY
 END;
 $BODY$;
 
-CREATE OR REPLACE FUNCTION storage.readallformultipledataelement(_instanceGuid UUID[])
```

Comment:
Do we need a DROP statement in our scripts?

Reply:
Then we need a new migration folder and script also. I planned to do it manually.

Reply:
okay..